catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/metacardtransformer/MetacardTransformerProducer.java

```diff
@@ -77,7 +78,8 @@ public void process(Exchange exchange) throws Exception {
     }
   }
 
-  private MetacardTransformer lookupTransformerReference(String metacardTransformerId) {
+  @VisibleForTesting
+  MetacardTransformer lookupTransformerReference(String metacardTransformerId) {
     Bundle bundle = FrameworkUtil.getBundle(this.getClass());
```

Comment:
❓Should we inject a `Function<Class<T>, Bundle> bundleLookup` and use `bundleLookup.apply(this.getClass())` here instead of making the whole method `@VisibleForTesting` and having to change its visibility? This would also help not decrease the code coverage.